[enterprise-4.6] GH#33682: Updating the disk encryption section (#34427)
openshift/openshift-docs: modules/installation-special-config-encrypt-disk-tpm2.adoc

[id="installation-special-config-encrypt-disk-tpm2_{context}"]
= Enabling TPM v2 disk encryption

> Use this procedure to enable TPM v2 mode disk encryption during an {product-title} installation.

Reviewer: ```suggestion
Use the following procedure to enable TPM v2 mode disk encryption during an {product-title} installation.
```

Author: Resolved.

.Prerequisites

> * You downloaded the {product-title} installation program on your installation node.

Reviewer: ```suggestion
* You have downloaded the {product-title} installation program on your installation node.
```
For consistency sake with the bullet below.

Author: Resolved.

        mode: 420
        overwrite: true
        path: /etc/clevis.json
----

> . Make a backup copy of the YAML files. You should do this because the manifest files will be consumed when you create the Ignition config files.

Reviewer: ```suggestion
. Create a backup copy of the YAML files. The original YAML files are consumed when you create the Ignition config files.
```

Author: Resolved.
